docs/index.rst

```diff
@@ -70,6 +70,13 @@ After installing MXNet, you can install the GluonNLP toolkit by
    pip install gluonnlp
 
 
+.. hint::
+
+   For more detailed guide on installing pre-release from latest master branch,
```

Comment:
I didn't call it nightly since we don't actually have a nightly tag. Hope that's ok.